lib/generators/sufia/templates/workflow.json.erb

```diff
@@ -9,19 +9,45 @@
                     "name": "deposit",
                     "from_states": [],
                     "transition_to": "pending_review",
+                    "attributes": {
+                        "presentation_sequence": 1
+                    },
                     "notifications": [
                         {
                             "notification_type": "email",
                             "name": "Sufia::Workflow::PendingReviewNotification",
                             "to": ["approving"],
                             "cc": []
                         }
+                    ],
+                    "methods": [
+                        "CurationConcerns::Workflow::DeactivateObject"
                     ]
-                },
-                {
+                }, {
+                    "name": "depositor_review",
+                    "from_states": [{"names": ["complete", "pending_review"], "roles": ["depositor_reviewing"]}],
```

Comment:
I believe you should be able to use the same role here as for the `approve` action, since the same users will be either approving or requesting changes for a deposit. That role is `approving`.